### ISSUE TITLE
typehint $app in routes.php and pipeline.php

### ISSUE THREAD

**Issue by belgattitude** [zendframework/zend-expressive-skeleton#154]
Just a tiny addition in config files that helps code completion.

Can be useful to typehint $app in routes.php and pipeline.php like this:

```php
/** @var \Zend\Expressive\Application $app */
```

I'll make a P/R soon.

**Comment by belgattitude** [comment]
Side note for people using phpstan.

To prevent phpstan to complain, simply create the `phpstan.neon` file and add

```neon
parameters:
	reportUnmatchedIgnoredErrors: false
	ignoreErrors:
		# For zend-expressive tests, the routes.php and pipeline.php
		# will break with phpstan.
		- '#Undefined variable: \$app#'
```

and refer it with

```shell
$ vendor/bin/phpstan analyse -l 5 -c phpstan.neon src tests
```

**Comment by xtreamwayz** [comment]
Merged with #155 into develop.
